hal/src/nRF52840/exflash_hal_params.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2022 Particle Industries, Inc.  All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation, either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <stdint.h>
+#include "nrfx_qspi.h"
+
+#define MX25R6435F_OTP_SECTOR_SIZE      1024
+#define MX25L3233F_OTP_SECTOR_SIZE      512
+#define GD25_SECURITY_REGISTER_COUNT    3
+#define GD25_SECURITY_REGISTER_SIZE     1024
+#define GD25_OTP_SECTOR_SIZE            (GD25_SECURITY_REGISTER_SIZE * GD25_SECURITY_REGISTER_COUNT)
```

Comment:
Should we reverse `GD25_OTP_SECTOR_SIZE` and `GD25_SECURITY_REGISTER_SIZE`? The way we define the sector size is different from it in our cognition.